src/pygccxml/parser/config.py

```diff
@@ -451,35 +452,20 @@ def create_compiler_path(xml_generator, compiler_path):
     if xml_generator == 'castxml' and compiler_path is None:
         if platform.system() == 'Windows':
             # Look for msvc
-            compiler_path = __get_first_compiler_in_path('where', 'cl')
+            compiler_path = shutil.which('cl')
```

Comment:
Was this tested on Windows?

Reply:
I don't have a Windows system to test but according to Python docs, it should work.

Reply:
I think it should be fine on Windows too.  `shutil.which` is probably doing even more complex things than just calling `where` like we did.